tests/integration_tests/bugs/test_lp1900837.py

```diff
@@ -0,0 +1,28 @@
+"""Integration test for LP: #1900836.
+
+This test mirrors the reproducing steps from the reported bug: it changes the
+permissions on cloud-init.log to 600 and confirms that they remain 600 after a
+reboot.
+"""
+import pytest
+
+
+def _get_log_perms(client):
+    return client.execute("stat -c %a /var/log/cloud-init.log")
+
+
+@pytest.mark.sru_2020_11
+class TestLp1900836:
+    def test_lp1900836(self, client):
```

Comment:
I kindof like that bugid is encoded in test name here as I think that makes those tests seachable via `-k bugid` I think. But I don't know if we need that at class level and test level or how we want to organize naming conventions vs marks

Reply:
> I kindof like that bugid is encoded in test name here as I think that makes those tests seachable via `-k bugid` I think. But I don't know if we need that at class level and test level or how we want to organize naming conventions vs marks

+1 on this, I'm going to do some rejigging to only capture bug number in one part of the test's "full" name, so we can still select on bug number but don't just have "lp1900837" repeated ad nauseum.